steps/common.py

```diff
@@ -1,7 +1,117 @@
 '''Common test methods'''
 
+import os
+import re
+
 from behave import *
 
+
+def exec_service_cmd(context, action, service_name, host=None):
+    if host == None:
+        host = context.target_host
+
+    #cmd = context.service_cmd
+    cmd = "systemctl ACTION SERVICE_NAME"
+    cmd = re.sub("ACTION", action, cmd)
+    cmd = re.sub("SERVICE_NAME", service_name, cmd)
+
+    result = context.remote_cmd(cmd='command',
+                                host=host,
+                                module_args=cmd,
+                                ignore_rc=True)
```

Comment:
Instead of using a command here, it would be nicer to use ansible's [service module](http://docs.ansible.com/ansible/service_module.html).